config/default.yml

```diff
@@ -783,6 +783,11 @@ PrePush:
     description: 'Running RSpec test suite'
     required_executable: 'rspec'
 
+  Minitest:
+    enabled: false
+    description: 'Running Minitest test suite'
+    command: ['bundle', 'exec', 'rake', 'test']
```

Comment:
Under the [Running Your Tests](https://github.com/seattlerb/minitest/blob/9fa5fa541528e5e74bdf4ee0c7692f5e45ace8c5/README.txt#L242) section of the documentation it mentions you can execute the tests directly.

In this case, setting the `required_library` option for the hook to `minitest` and updating the `command` option to `['ruby', '-Ilib:test', 'test']` as mentioned in the documentation.

Basically, invoke the tool directly rather than indirectly, and this pull request is good to go.